app/api/routes.py

```diff
@@ -245,20 +245,44 @@ def search_results():
 
     # Fetch the filter params from the url, if they were provided.
     paid = request.args.get('paid')
-    filters = ''
+    category = request.args.get('category')
+    language = request.args.get('language')
+    filters = []
 
     # Filter on paid
     if isinstance(paid, str):
         paid = paid.lower()
         # algolia filters boolean attributes with either 0 or 1
         if paid == 'true':
-            filters += 'paid=1'
+            filters.append('paid=1')
         elif paid == 'false':
-            filters += 'paid=0'
+            filters.append('paid=0')
+
+    # Filter on category
+    if isinstance(category, str):
+        filters.append(
+            f"(category:{category} OR category:{category.lower()}"
+            f" OR category:{category.capitalize()})"
+        )
+
+    # Filter on language
+    if isinstance(language, str):
+        filters.append(
+            f"(languages:f{language} OR languages:{language.lower()}"
+            f" OR languages:{language.capitalize()})"
+        )
 
     try:
+        # See algolia.com/doc/api-reference/api-parameters/attributesForFaceting/
+        index.set_settings({
```

Comment:
This really seems like something we should only be doing once, not on every search request. If it is to be called more than once, it seems like it's meant to be set before indexing the resource in Algolia in `create_resource()`. I could be wrong on how this works though. Can you point out the part of the documentation that makes you believe this should be called in this particular place in the integration?

Reply:
I went ahead and added it in the dashboard. Please remove this method call

<img width="687" alt="Screen Shot 2019-10-14 at 3 41 57 PM" src="https://user-images.githubusercontent.com/2008701/66781568-3c43e400-ee99-11e9-82a0-001106f19aa4.png">
.